code/aepuitemplates/src/main/java/com/adobe/marketing/mobile/aepuitemplates/SmallImageTemplate.kt

```diff
@@ -0,0 +1,26 @@
+package com.adobe.marketing.mobile.aepuitemplates
+
+import com.adobe.marketing.mobile.aepuitemplates.utils.AepUiTemplateType
+
+/**
+ * Class representing a small image template, which implements the [AEPUITemplate] interface.
+ *
+ * This class contains properties for an image URL, title, and description.
+ */
+class SmallImageTemplate : AEPUITemplate {
+    /** The URL of the image for the small image template. */
+    val imageUrl: String = ""
+
+    /** The title for the small image template. */
+    val title: String = ""
+
+    /** The description for the small image template. */
+    val description: String = ""
+
+    /**
+     * Returns the type of this template, which is [AepUiTemplateType.SMALL_IMAGE].
+     *
+     * @return A string representing the type of the template.
+     */
+    override fun getType() = AepUiTemplateType.SMALL_IMAGE.typeName
```

Comment:
Wondering if we should change `getType` to return `AepUiTemplateType` instead of string. Thoughts?